backend/debug_routes.py

```
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+from fastapi import FastAPI
+from app.api.routes import api_router
+
+def debug_routes():
+    """Debug the registered routes."""
+    app = FastAPI()
+    app.include_router(api_router, prefix="/api/v1")
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)